CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
do we need to capture - if there is a conflict how would that be resolved ?

Reply:
Conflict resolution is handled by TSC. This is covered in the contribution guide.